manuscript/recipes/duplicati.md

```diff
@@ -0,0 +1,115 @@
+hero: Duplicati - Yet another option to backup your exciting stuff. It's good to have options.
+
+# Duplicati
+
+Intro
+
+![duplicati Screenshot](../images/duplicati.jpg)
+
+
+[Duplicati](https://www.duplicati.com/) is a free and open-source backup software to store encrypted backups online For Windows, macOS and Linux (our favorite, yay!).
+
+Similar to the other backup solution options in the Cookbook, you can use Duplicati to backup all our data-at-rest to a wide variety of locations, including, but not limited to:
+
+- FTP servers
+- SSH servers
+- WebDAV endpoints
+- Backblaze B2
+- Tardigrade
+- Microsoft OneDrive
+- Amazon S3
+- Google Drive
+- box.com
+- Mega
+- hubiC
+- many others
+
+!!! note
+    Since Duplicati itself offers no user authentication, this design secures Duplicati behind a [Traefik Forward Auth proxy](/ha-docker-swarm/traefik-forward-auth), so that in order to gain access to the Duplicati UI at all, oauth2 authentication (_to GitHub, GitLab, Google, etc_) must have already occurred.
```

Comment:
This presumably came from an older recipe which used oauth_proxy :) Without this background, the user may be confused. Try rewording to remove the reference to "proxy" and "oauth2", since neither are strictly required for TFA anymore

Reply:
Fixed (I think). Let me know if that's not what you were expecting.